python/railjson_generator/pyproject.toml

```diff
@@ -1,41 +1,12 @@
 [project]
 name = "railjson_generator"
 version = "0.4.9"
-
-[tool.poetry]
-name = "railjson_generator"
-version = "0.4.9"
 description = ""
-authors = ["OSRD <contact@osrd.fr>"]
-
-[tool.poetry.dependencies]
-osrd-schemas = { path = "../osrd_schemas/", develop = false }
-pytest = "^7.4.3"
-pytest-cov = "^4.1.0"
-python = ">=3.9,<3.12"
-
-[tool.poetry.group.dev.dependencies]
-black = "^24.4.2"
-flake8-formatter-junit-xml = "^0.0.6"
-isort = "^5.12.0"
-pyproject-flake8 = "^6.0.0.post1"
-pytype = { platform = "linux", version = "^2023.10.17" }
-
-[tool.flake8]
-ignore = "W503,E203"
-max-line-length = 120
-exclude = [".venv/"]
-
-[tool.black]
-line-length = 120
```

Comment:
Why not keep the line length of 120 ? It seems to be the norm throughout our repos (and it's more pleasant to use in my opinion)

Reply:
Because `black` is not there anymore and because I like not customizing the formatter to avoid subjective debat 🤷 But if I'm the only one, [it can be put back in `ruff`'s configuration](https://docs.astral.sh/ruff/settings/#line-length).

Reply:
OK, I feel free 🆓